distributed/shuffle/tests/test_shuffle.py

```diff
@@ -1055,63 +1064,66 @@ def __init__(self, value: int) -> None:
 
     counter = count()
     workers = ["a", "b", "c"]
-    npartitions = 5
```

Comment:
I'm not sure if I understand your intentions correctly, but `npartitions` is the number of output partitions. This entire test only has one input partition, which is marked by a hard-coded `0` here: https://github.com/dask/distributed/blob/e30a3e4df37c1b0f43d1b71c7434216f3bcdc440/distributed/shuffle/tests/test_shuffle.py#L1175-L1176

Reply:
oh well, you are right. I tested it before on tets_basic_integration but I found it to be too slow

Reply:
`test_basic_integration` has 8640 rows per partition. We could reduce that if we wanted to increase the number of partitions significantly. (I'm also open to reducing the size in general or refactoring those tests in general, but I don't see much value in that at the moment.)